examples/custom-output-paths/keystone.ts

```diff
@@ -0,0 +1,19 @@
+import { config } from '@keystone-6/core';
+import { fixPrismaPath } from '../example-utils';
+import { lists } from './schema';
+
+export default config({
+  db: {
+    provider: 'sqlite',
+    url: process.env.DATABASE_URL || 'file:./keystone-example.db',
+
+    // WARNING: this is only needed for our monorepo examples, dont do this
+    ...fixPrismaPath,
+  },
+  lists,
+
+  // TODO: add explanation about why you might want to do this
```

Comment:
// specifying a custom types path may be needed in monorepos or with alternative package managers